### DISCUSSION TITLE
Bug: PNPM Install Errors

### DISCUSSION THREAD

**Discussion by TheGreatAxios** [wevm/ox#31] [Q&A]
### Check existing issues

- [X] I checked there isn't [already an issue](https://github.com/wevm/ox/issues) for the bug I encountered.

### Ox Version

0.1.1

### Current Behavior

pnpm install fails during `node-gyp rebuild`

### Expected Behavior

Should install 

### Steps To Reproduce

1. Followed contributing.md
2. During install on Mac with valid Node + PNPM Versions, receving node-gyp build errors
3. Recieve Output -> https://app.warp.dev/block/5ozafJenfbDvMQqTBRTFVY

### Anything else?

MacOS - Sequoia 15.0.1
Tested with Node Verisons - v22.8.0 & v22.3.0
Tested with PNPM Versions -> v9.7.1 & 9.12.3

**Comment by tmm** [comment]
Can you try the following:

```bash
sudo rm -rf /Library/Developer/CommandLineTools
xcode-select --install
pnpm i
```

Might also be worth trying Node v20.
